data/channels.csv

```diff
@@ -17875,7 +17876,7 @@ PloiestiTV.ro,Ploiesti TV,,,,RO,,,c/RO,ron,,FALSE,,,,,https://i.imgur.com/Y3p5gh
 PlovdivskaPravoslavnaTV.bg,Plovdivska Pravoslavna TV,,,,BG,,,c/BG,bul,,FALSE,,,,http://www.plovdivskamitropolia.bg/,https://i.imgur.com/BuN0Osz.png
 PLP.gr,PLP,,,,GR,,,c/GR,ell,,FALSE,,,,,https://i.imgur.com/N4SY7rr.png
 PLTV.ir,PLTV,,,,IR,,,c/IR,fas,movies,FALSE,,,,https://www.parsilandtv.com/,https://i.imgur.com/heWo9rY.png
-PlugRTL.be,Plug RTL,,,,BE,,,c/BE,fra,,FALSE,2004-02-13,,,http://www.rtl.be/tv/plugrtl/,https://i.imgur.com/EgqIM6M.png
+PlugRTL.be,Plug RTL,,,,BE,,,c/BE,fra,,FALSE,2004-02-13,,,http://www.rtl.be/tv/plugrtl/,https://i.imgur.com/aFnOCWV.png

```

Comment:
The old version of the logo is bigger than the new one.

Reply:
Apologies

This is a clearer and larger version (512*196)

https://i.imgur.com/uSKrgLt.png

Reply:
What's wrong with the old version?

https://i.imgur.com/EgqIM6M.png

Reply:
The white space in "P" and "R"; newer logo is darker, better contrast.
![EgqIM6M~2](https://user-images.githubusercontent.com/61802643/216491834-e5cfa4c6-60f2-4ec5-b005-91a26d600dfd.png)

Reply:
Now I see.

Reply:
```suggestion
PlugRTL.be,Plug RTL,,,,BE,,,c/BE,fra,,FALSE,2004-02-13,,,http://www.rtl.be/tv/plugrtl/,https://i.imgur.com/uSKrgLt.png
```

Reply:
Perfect

Reply:
You need to update your PR so I can approve it.

Reply:
Ok will do, thanks